docs/notebooks/run_model_with_different_dataset_on_CRIB.ipynb

```diff
@@ -0,0 +1,317 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running the STEMMUS_SCOPE model with different dataset on CRIB using MATLAB\n",
+    "\n",
+    "This notebook shows how to run the STEMMUS_SCOPE model uisng source code, including preprocessing and postprocessing. For postprocesisng, the file required_netcdf_variables.csv can be found at https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/main/utils/csv_to_nc/required_netcdf_variables.csv.\n",
+    "\n",
+    "## Which options are available?\n",
+    "\n",
+    "See the [Run the model with different options](./run_model_with_different_options.ipynb) for avialbale options e.g. Executable file, Matlab, Octave.\n",
```

Comment:
I am not sure about calling it "options" because that sounds more like different model configurations to me.

Perhaps calling saying "depending on your system (OS, Matlab availability, etc.), there are a few ways you can run the model"